mirny.py

```diff
@@ -153,6 +153,40 @@ def connect_ext(self, ext, adr, mask):
             ),
         ]
 
+    def connect_almazny_passthrough(self, pt, adr, mask):
+        # for use w/ almazny
+        self._check_intersection(adr, mask)
+        self._slaves.append((pt, adr, mask))
+        stb = AsyncRst()
+        self.submodules += stb
+
+        saved_adr = Signal(2)
+        latch_signal = Signal()
+
+        self.comb += [
+            stb.i.eq(((self.bus.adr & adr) == adr) & (self.ext.cs)),
+            stb.ce.eq(self.bus.re),
+            pt.mosi.eq(self.ext.sdi & stb.o),
+            pt.clk.eq(self.ext.sck & stb.o),
+            # simple solution - latch after writing is done
+            latch_signal.eq(~stb.o),
+        ]
+
+        # update saved address when we write to almazny
+        self.sync += [
+            If(stb.o,
+                saved_adr.eq(self.bus.adr[:2]),
+            )
+        ]
+
+        # latch can stay up indefinitely, it's only rising edge that counts
+        self.comb += [
```

Comment:
Better to decode this not comb but in the `sync` block above to avoid transients.
Why not four AsyncRst to do the decoding and latching? Why does this need to differ from the existing code in `connect_ext` so much?

Reply:
I understand the concerns, and you're right about the transients. I don't think AsyncRsts are necessary (...or maybe I don't know how to use them efficiently here), so I got it working, in the ``sync`` block, using four Signals instead.

Reply:
`AsyncRst` are exactly what you want. Just compare your application with the existing attenuators (0x8-0xb).

Reply:
`AsyncRst` are exactly what you want. Just compare your application with the existing attenuators (0x8-0xb).